climakitae/selectors.py

```diff
@@ -361,3 +361,49 @@ def _display_select(selections, location, location_type="area average"):
         )
     )
     return pn.Column(first_row, location_chooser)
+
```

Comment:
nit: would prefer a less generic name like file_out_selector.py to avoid potential redundancies in the future

Reply:
oh so a separate file for export-related selectors?

Reply:
I think it's good to put all of these kinds of UI things in one file together...